script/bootstrap

```diff
@@ -47,6 +47,13 @@ if ! type -p http &>/dev/null; then
     python -m pip install --upgrade pip setuptools
     # Install HTTPie
     python -m pip install --upgrade httpie
+    # If not HTTPie is not installed above, lets try:
+    # Attempt #1: install without multidict compilation
+    [[ -z $(pip show httpie 2>/dev/null) ]] && MULTIDICT_NO_EXTENSIONS=1 python -m pip install --no-cache-dir httpie
+    # Attempt #2: install older version that does not require multidict
+    [[ -z $(pip show httpie 2>/dev/null) ]] && python -m pip install httpie==2.6.0
+    # Return code for $ERROR_CODE
+    python -m pip show httpie >/dev/null 2>&1
```

Comment:
Just out of curiosity, why use `python -m pip show httpie` here? For consistency, I might prefer sticking to `type -p http`, but I’m open to other ideas if you think this is a better approach.

Reply:
I think `pip show` determines more accurately if `httpie` was installed. It is worth mentioning (and wasn't considered when submitting PR) that it performs a lot slower than `type`

```sh-session
$ time pip show httpie
real    0m2.190s
user    0m0.703s
sys     0m1.139s

$ time pip show doesnotexist
real    0m2.178s
user    0m0.671s
sys     0m1.077s

$ time type -p http
real    0m0.009s
user    0m0.000s
sys     0m0.000s

$ time type -p doesnotexist
real    0m0.015s
user    0m0.000s
sys     0m0.016s
```